.github/ISSUE_TEMPLATE/studentProgram.md

```diff
@@ -9,21 +9,43 @@ If it is a general question / idea of NNI, you could just make a short summary.
  
 If it is a operational issue, please fill operational issue template and provide as many details as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!-->
  
-## 问题概述
+## NNI 学生项目问题概述（中文版）
 
 **请简要概述您的问题 / 观点** :
 
 
+**请提供 NNI 环境信息** :
+
```

Comment:
can wen directly merge the chinese and english together instead of putting them in 2 parts?
i.e. 
**请简要概述您的问题 / 观点**  :
**_Short summary about the question / idea_** :

Reply:
Thanks for your review~I made some minors here.

Guides (with <!....>) before "NNI 学生项目问题概述":

下面是 NNI 学生项目问题表单，我们鼓励您提出问题或分享观点，期待同学们的思维碰撞和灵感火花!（中英文均可）
Here is an issue template for NNI student program China. You are encouraged to raise concerns about any issue and share your ideas of NNI or our student program. Both Chinese and English are acceptable.

如果是概念性问题，您可以进行简单概述；
If it is a general question / idea of NNI, please make a short summary.

如果是操作性问题，您需要尽可能详细地提供 NNI 环境信息。
If it is a operational issue, please fill nni environment and provide as many details as possible. 

## NNI 学生项目问题概述 / General Question of Student Program

****请简要概述您的问题 / 观点** :
**Short summary about the question / idea** :**

请提供 NNI 环境信息:
nni Environment:

nni version:
nni mode(local|pai|remote):
OS:
python version:
is conda or virtualenv used?:
is running in docker?:
nnimanager.log and dispatcher.log:
nnictl stdout and stderr:

## 其他建议 / Other Advice

**是否需要更新文档（是 / 否）** :
**Need to update document ( yes / no )** :

**其他分享内容** :
**Anything else we need to know** :